src/scenes/AcceleratorRouter/Overview/index.tsx

```diff
@@ -6,15 +6,14 @@ import { makeStyles } from '@mui/styles';
 import { Tabs } from '@terraware/web-components';
 
 import Page from 'src/components/Page';
-import Button from 'src/components/common/button/Button';
-import { APP_PATHS } from 'src/constants';
 import { useLocalization, useUser } from 'src/providers';
 import ParticipantsList from 'src/scenes/AcceleratorRouter/Participants/ParticipantsList';
 import strings from 'src/strings';
-import useDeviceInfo from 'src/utils/useDeviceInfo';
 import useQuery from 'src/utils/useQuery';
 import useStateLocation, { getLocation } from 'src/utils/useStateLocation';
 
+import CohortsListView from '../Cohorts/CohortsListView';
```

Comment:
Use absolute path for directories not within this file's hierarchy?

Reply:
Oops missed this one, thanks!